src/dev/impl/DevToys/ViewModels/Tools/Formatters/XmlFormatter/XmlFormatterToolProvider.cs

```diff
@@ -0,0 +1,48 @@
+﻿#nullable enable
+
+using System.Composition;
+using DevToys.Shared.Api.Core;
+using DevToys.Api.Tools;
+using DevToys.Helpers;
+using DevToys.Core.Threading;
+using Windows.UI.Xaml.Controls;
+
+namespace DevToys.ViewModels.Tools.Formatters.XmlFormatter
+{
+    [Export(typeof(IToolProvider))]
+    [Name("Xml Formatter")]
+    [Parent(FormattersGroupToolProvider.InternalName)]
+    [ProtocolName("xmlformat")]
+    [Order(0)]
+    [NotScrollable]
+    internal sealed class XmlFormatterToolProvider : ToolProviderBase, IToolProvider
+    {
+        private readonly IMefProvider _mefProvider;
+
+        public string MenuDisplayName => LanguageManager.Instance.XmlFormatter.MenuDisplayName;
+
+        public string? SearchDisplayName => LanguageManager.Instance.XmlFormatter.SearchDisplayName;
+
+        public string? Description => LanguageManager.Instance.XmlFormatter.Description;
+
+        public string AccessibleName => LanguageManager.Instance.XmlFormatter.AccessibleName;
+
+        public TaskCompletionNotifier<IconElement> IconSource => CreateFontIcon("\uf2ef");
```

Comment:
This icon is very close from the one of HTML Encoder / Decoder. It would be nice to differentiate these too icons, although both make sense for HTML and XML. We can try to find a different one for XML and / or HTML.

What do you think of something like this for HTML?

![image](https://user-images.githubusercontent.com/3747805/151490746-955f0d6e-91bc-419d-b4cd-5259fb9bf815.png)

Reply:
Looking good to me: 
![image](https://user-images.githubusercontent.com/5731724/151673057-541e2d8b-5839-451d-97f7-b52309fd2200.png)